OrbitClientData/include/OrbitClientData/FunctionInfoSet.h

```diff
@@ -42,4 +42,9 @@ using FunctionInfoSet =
     absl::flat_hash_set<orbit_client_protos::FunctionInfo, internal::HashFunctionInfo,
                         internal::EqualFunctionInfo>;
 
+template <class V>
+using FunctionInfoMap =
+    absl::flat_hash_map<orbit_client_protos::FunctionInfo, V, internal::HashFunctionInfo,
+                        internal::EqualFunctionInfo>;
+
```

Comment:
We need to be careful here - we should probably keep module_path as part of FunctionInfo forever to avoid unwanted conflict.

Also seems like file and line should not be part of the equals/hash<> operators.

Reply:
Yes, I agree that we should keep module_path as part of function info.
Regarding the equals/hash, I would say to use all the fields that are in the proto. Ideally a proto generated class should have equals/hash to be used in sets/maps, but this will only come in the future.